examples/parse_time.py

```diff
@@ -0,0 +1,94 @@
+"""
+================================================
+Parsing times with sunpy.time.parse_time
+================================================
```

Comment:
I forget if Github uses a monospace font but the `=` should be the same length as the title.